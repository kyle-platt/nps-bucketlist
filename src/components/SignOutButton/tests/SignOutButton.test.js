import React from 'react';
import renderer from 'react-test-renderer';
import { SignOutButtonTest } from '../SignOutButton';

describe('SignOutButton Tests', () => {
    it('renders with some data', () => {
        const props = {
            firebase: {
                doSignOut: () => {},
            }
        };
        const tree = renderer.create(
            <SignOutButtonTest {...props} />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
