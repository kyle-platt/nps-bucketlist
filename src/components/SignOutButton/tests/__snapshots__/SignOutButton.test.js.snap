// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignOutButton Tests renders with some data 1`] = `
<button
  onClick={[Function]}
  type="button"
>
  Sign Out
</button>
`;
